doc/tutorials/elektra-merge-integration.md

```diff
@@ -1,104 +0,0 @@
-# How-To: Integrate elektra-merge Into a Debian Package
```

Comment:
@Chemin1 we will remove this for now, could you please write a similar tutorial?

Reply:
@markus2330  Yes. I opened a new issue and added it to the project so that the task is not forgotten.

Reply:
Thank you! I hope you had nice holidays!